src/EditRow.js

```diff
@@ -0,0 +1,62 @@
+import React, { useState } from 'react';
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+
+const EditRow = ({ editInputData, handleEditFormData , handleCancelClick}) => {
+  const [startDate] = useState(new Date());
+
+  return (
+    <tr>
+      <td>
+        <input
+          type='text'
+          required='required'
+          placeholder='Enter a name...'
+          name='name'
+          value={editInputData.name}
```

Comment:
I guess we can destructure the editInputData above
Something like
const { name, email, birthdate, gender, education } = editInputData